UI/win-update/updater/updater.cpp

```diff
@@ -1564,6 +1565,32 @@ static bool Update(wchar_t *cmdLine)
 					   source.c_str(), size);
 	}
 
+	/* ------------------------------------- *
+	 * Deduplicate Downloads                 */
+
+	vector<BYTE *> hashes;
```

Comment:
Are we able to use something like `unordered_set` instead of `vector`? We don't care about the actual values, only existence, so it should provide constant-time performance instead of iterating over everything.

Reply:
Updated it to simply use the `tempPath` with an unordered set since `std::hash` contains an implementation for `wstring` so no additional code or conversion necessary.